plugins/woocommerce/tests/php/src/Blocks/StoreApi/Routes/Patterns.php

```diff
@@ -26,7 +26,7 @@ public function setUp(): void {
 	public function test_post_endpoint_when_tracking_is_not_allowed() {
```

Comment:
Shall we add a test to cover `throws`?

Reply:
I've been looking into this and I think is not good to throw that exception from the endpoint, it was resulting in a data error, I've refactored it to return an error and added tests for the `GET` endpoint.
Also, I moved the tests to the new appropriate location.
https://github.com/woocommerce/woocommerce/pull/50400/commits/3cc4e76e0092f294f05a5b5d1c1a1e3a7c9dffc2